packages/svelte/src/runtime/internal/Component.js

```diff
@@ -157,23 +159,23 @@ export let SvelteElement;
 
 if (typeof HTMLElement === 'function') {
 	SvelteElement = class extends HTMLElement {
-		$$componentCtor;
-		$$slots;
-		$$component;
-		$$connected = false;
-		$$data = {};
-		$$reflecting = false;
+		$$ctor;
```

Comment:
I would put comments next to each of these with their old names or describing what they hold as it might be hard to figure out from just the single-letter abbreviation

Reply:
Preferably JSDoc comments above so that they show up when hovering over where the properties are used.